docs/usage/pivoting.md

```diff
@@ -1,73 +1,80 @@
 # Pivot and investigate
 
-In Opencti, all data can be represented as a large knowledge graph: everything is linked to something. 
-You can pivot on any entity and on any relationship you have in your platform, using investigations.
+In OpenCTI, all data are structured as an extensive knowledge graph, where every element is interconnected. The investigation functionality provides a powerful tool for pivoting on any entity or relationship within the platform. Pivoting enables users to explore and analyze connections between entities and relationships, facilitating a comprehensive understanding of the data.
 
-Investigations are available on the top right of the top bar:
+To access investigations, navigate to the top right corner of the toolbar:
 
 ![Top menu investigation](assets/top-menu-investigation.png)
 
-Investigations are organized by workspace. When you create a new empty workspace, it will only be visible by you and enables you to work on your investigation before sharing it.
+!!! note "Access restriction"
 
-In your workspace, you can add entities that you want to investigate, visualize the data linked to these entities, add relationships, and export your investigation graph in pdf, image or as new stix report.
+    When an investigation is created, it is initially visible only to the creator, allowing them to work on the investigation before deciding to share it. The sharing mechanism is akin to that of dashboards. For further details, refer to the [Access control section](dashboards.md#access-control-section) in the dashboard documentation page.
```

Comment:
You mean, put a "s" to "creator" because I use "them"?
If so, it's not necessarily wrong (here, it's intentional). When it can be a man or a woman (so "him" or "her"), it's correct to use "them".

Reply:
Oh ok, i didn't understand it in that way, so it's ok never mind